ui/src/components/executions/ServiceInfo.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <component
+        :is="component"
+        v-if="service != null"
+    >
+        <template #default>
+            <strong>{{ service.id }}</strong>: {{ $t("hostname") }}={{ service.server.hostname }}, {{ $t("version") }}={{ service.server.version }}, {{ $t("state") }}={{ service.state }}
+        </template>
+    </component>
+</template>
+
+<script>
```

Comment:
```suggestion
<script setup lang="ts">
```

Same here as below, please move the component to composition API

Reply:
None of the components of the execution page has been migrated so I'l leaving that for a later refactoring.